docs/en/setup/service-agent/java-agent/Application-toolkit-tracer.md

```diff
@@ -0,0 +1,213 @@
+# Create Span
+
+* Use `Tracer.createEntrySpan()` API to create entry span, and then use `SpanRef` to contain the reference of created span in agent kernel. The first parameter is operation name of span and the second parameter is the `ContextCarrierRef` instance which is the reference of contextcarrier in agent kernel. If the second parameter is not null, the process of creating entry span will do the extract operation which will be introduced in **inject/extract** scenario.
+
+  ```java
+  import Tracer;
```

Comment:
This import seems wrong? Why here without package name?

Reply:
I find that the old doc of tracing API only imports the class name, so I just write the Tracer class. So it should import the entire package, "org.apache.skywalking.apm.toolkit.trace.Tracer"?

Reply:
But actually, I find lots of package importing are lost in the Application-toolkit-tracer.md .